### PR TITLE
Backend: Add PR review policy

### PR DESCRIPTION
**Pull request by lineargraph**
## PR Reviews

When your PR is marked as ready for review, some of our maintainers will look through your code to make sure everything is good to go. In order to do this, they may request some changes you will need to do, **or fix smaller stuff (like merge conflicts) for you**. If a maintainer has reviewed your PR, make sure to **pull any of their changes** into your local project before doing more work on your code. Having maintainers fix small stuff for you helps us speed up the process of merging your PR, so if some of your systems warrant further care, be sure to let us now (preferably with a code comment).

Make sure to only mark your PR as "Ready to review" when it is. You can keep a draft PR open until then.

## What
Describe what this pull request does, including technical details, screenshots, links to discord, etc.

<details>
<summary>Images</summary>

</details>

## Changelog Technical Details
+ Added PR review policy. - nea
    * This should help us speed up the PR review process.

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <!-- remove all unused parts -->
 
+## PR Reviews
+
+When your PR is marked as ready for review, some of our maintainers will look through your code to make sure everything is good to go. In order to do this, they may request some changes you will need to do, **or fix smaller stuff (like merge conflicts) for you**. If a maintainer has reviewed your PR, make sure to **pull any of their changes** into your local project before doing more work on your code. Having maintainers fix small stuff for you helps us speed up the process of merging your PR, so if some of your systems warrant further care, be sure to let us now (preferably with a code comment).
```
Comment (jani270) [inline pull_request_template.md]:
```suggestion
When your PR is marked as ready for review, some of our maintainers will look through your code to make sure everything is good to go. In order to do this, they may request some changes you will need to do, **or fix smaller stuff (like merge conflicts) for you**. If a maintainer has reviewed your PR, make sure to **pull any of their changes** into your local project before doing more work on your code. Having maintainers fix small stuff for you helps us speed up the process of merging your PR, so if some of your systems warrant further care, be sure to let us know (preferably with a code comment).
```

Comment (hannibal002) [reply pull_request_template.md]: +1 this

**Review by hannibal002 — APPROVED** [review]

File: pull_request_template.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 <!-- remove all unused parts -->
 
+## PR Reviews
+
+When your PR is marked as ready for review, some of our maintainers will look through your code to make sure everything is good to go. In order to do this, they may request some changes you will need to do, **or fix smaller stuff (like merge conflicts) for you**. If a maintainer has reviewed your PR, make sure to **pull any of their changes** into your local project before doing more work on your code. Having maintainers fix small stuff for you helps us speed up the process of merging your PR, so if some of your systems warrant further care, be sure to let us know (preferably with a code comment).
+
+Make sure to only mark your PR as "Ready to review" when it is. If you still want to do major changes, you can keep a draft PR open until then.
```
Comment (RayDeeUx) [inline pull_request_template.md:7]:
```suggestion
Make sure to only mark your PR as "Ready to review" when it is. If you still want to do major changes, you can mark your PR as a draft until then.
```

**Comment by RayDeeUx** [comment]
oh bruh

**Pull request merged**